### ISSUE TITLE
Neo Magazin Royale kann nicht geöffnet werden

### ISSUE THREAD

**Issue by theincogtion** [generia/plugin.video.zdf_de_2016#5]
**Plattform**
Win 8.1 x64
Kodi 17
ZDF Mediathek 2016 1.0.3

**Reproduzieren:**
ZDF Mediathek 2016 -> Shows A-Z -> Shows with N -> Neo Magazin Royale (Posts 101 Beiträge) 
Es wird kein Inhalt angezeigt.

**Log**
`10:19:00.717 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [Plugin] - Python-Version: C:\Program Files (x86)\Kodi\kodi.exe 2.7.11 (default, Jan 16 2016, 16:21:17) [MSC v.1900 32 bit (Intel)]
10:19:00.717 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/ ...
10:19:00.717 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [Mediathek] - Timer - parsing url='https://www.zdf.de/ZDFplayer/configs/zdf/zdf2016/configuration.json' ...
10:19:00.717 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [ConfigurationResource] - Timer - loading url='https://www.zdf.de/ZDFplayer/configs/zdf/zdf2016/configuration.json' ...
10:19:01.954 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [ConfigurationResource] - Timer - loading url='https://www.zdf.de/ZDFplayer/configs/zdf/zdf2016/configuration.json' ... done. [1237 ms]
10:19:01.955 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [Mediathek] - Timer - parsing url='https://www.zdf.de/ZDFplayer/configs/zdf/zdf2016/configuration.json' ... done. [1238 ms]
10:19:01.966 T:2596  NOTICE: ZDF Mediathek 2016 [1]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/ ... done. [1250 ms]
10:19:03.221 T:5960  NOTICE: ZDF Mediathek 2016 [2]: [Plugin] - Python-Version: C:\Program Files (x86)\Kodi\kodi.exe 2.7.11 (default, Jan 16 2016, 16:21:17) [MSC v.1900 32 bit (Intel)]
10:19:03.221 T:5960  NOTICE: ZDF Mediathek 2016 [2]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=ShowsAzPage ...
10:19:03.647 T:5960  NOTICE: ZDF Mediathek 2016 [2]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=ShowsAzPage ... done. [426 ms]
10:19:07.448 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [Plugin] - Python-Version: C:\Program Files (x86)\Kodi\kodi.exe 2.7.11 (default, Jan 16 2016, 16:21:17) [MSC v.1900 32 bit (Intel)]
10:19:07.448 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=RubricPage&rubricUrl=%2fsendungen-a-z%3fgroup%3dn ...
10:19:07.448 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - rubric-page: url='/sendungen-a-z?group=n', listType='None', listStart='-1', listEnd='-1'
10:19:07.448 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - Timer - parsing url='https://www.zdf.de/sendungen-a-z?group=n' ...
10:19:07.449 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricResource] - Timer - loading url='https://www.zdf.de/sendungen-a-z?group=n' ...
10:19:07.815 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricResource] - Timer - loading url='https://www.zdf.de/sendungen-a-z?group=n' ... done. [367 ms]
10:19:07.822 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - Timer - parsing url='https://www.zdf.de/sendungen-a-z?group=n' ... done. [374 ms]
10:19:07.822 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/wissen/nano' and teaser-title '[Posts: 858 Beiträge] nano' ...
10:19:07.870 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/comedy/neo-magazin-mit-jan-boehmermann' and teaser-title '[Posts: 101 Beiträge] NEO MAGAZIN Royale' ...
10:19:07.870 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/filme/neu-im-kino' and teaser-title '[Posts: 19 Beiträge] Neu im Kino' ...
10:19:07.870 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/serien/the-night-manager' and teaser-title '[Posts: 2 Beiträge] The Night Manager' ...
10:19:07.871 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/serien/nordlicht-moerder-ohne-reue' and teaser-title '[Posts: 6 Beiträge] Nordlicht - Mörder ohne Reue' ...
10:19:07.871 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [RubricPage] - redirecting to rubric-url  '/serien/notruf-hafenkante' and teaser-title '[Posts: 186 Beiträge] Notruf Hafenkante' ...
10:19:07.871 T:7280  NOTICE: ZDF Mediathek 2016 [3]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=RubricPage&rubricUrl=%2fsendungen-a-z%3fgroup%3dn ... done. [423 ms]
10:19:09.953 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [Plugin] - Python-Version: C:\Program Files (x86)\Kodi\kodi.exe 2.7.11 (default, Jan 16 2016, 16:21:17) [MSC v.1900 32 bit (Intel)]
10:19:09.953 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=RubricPage&rubricUrl=%2fcomedy%2fneo-magazin-mit-jan-boehmermann ...
10:19:09.954 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [RubricPage] - rubric-page: url='/comedy/neo-magazin-mit-jan-boehmermann', listType='None', listStart='-1', listEnd='-1'
10:19:09.954 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [RubricPage] - Timer - parsing url='https://www.zdf.de/comedy/neo-magazin-mit-jan-boehmermann' ...
10:19:09.954 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [RubricResource] - Timer - loading url='https://www.zdf.de/comedy/neo-magazin-mit-jan-boehmermann' ...
10:19:10.402 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [RubricResource] - Timer - loading url='https://www.zdf.de/comedy/neo-magazin-mit-jan-boehmermann' ... done. [448 ms]
10:19:10.403 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [RubricPage] - Timer - parsing url='https://www.zdf.de/comedy/neo-magazin-mit-jan-boehmermann' ... done. [450 ms]
10:19:10.403 T:6516  NOTICE: ZDF Mediathek 2016 [4]: [Plugin] - executing url=plugin://plugin.video.zdf_de_2016/?apiToken=d2726b6c8c655e42b68b0db26131b15b22bd1a32&pagelet=RubricPage&rubricUrl=%2fcomedy%2fneo-magazin-mit-jan-boehmermann ... done. [450 ms]
`

**Comment by generia** [comment]
Danke für die Info.

So wie es aussieht wird der Link hinter "ZDF Mediathek 2016 -> Shows A-Z -> Shows with N -> Neo Magazin Royale (Posts 101 Beiträge)" weitergeleitet zu "https://www.neo-magazin-royale.de/zdi/" und verläßt damit die ZDF Mediathek.

Wenn man auf zdf.de die Links verfolgt sieht man das Verhalten:
- Sendungen A-Z > N Link: https://www.zdf.de/sendungen-a-z?group=n
- NEO MAGAZIN Royale Link: https://www.zdf.de/comedy/neo-magazin-mit-jan-boehmermann
- Jetzt erfolgt eine Weiterleitung nach "https://www.neo-magazin-royale.de/zdi/"

Mal sehen, ob Weiterleitungen durch das Plugin erkennbar sind.
Es ist dann auch die Frage, wie man das darstellt.

Long story short: es ist in dem Sinne kein Bug - aber die Anzeige/Behandlung von Weiterleitungen muss nochmal überdacht werden.

Für Anregungen bin ich immer zu haben ...

Viele Grüße


PS: Unter "ZDF Mediathek 2016 -> Rubriken -> Rubrik - Comedy -> NEO MAGAZIN ROYALE" sind die Videos abrufbar. Oder einfach die Suchfunktion nutzen, da finden sich die Videos auch.

**Comment by theincogtion** [comment]
Das er zu zdi weiterleitet hab ich auch bemerkt im log.
Mein workaround war folgender:
Ich bin auf die normale mediathek gegangen und habe den download link per yatse an kodi geschickt.
Vielleicht könnte man die zdi url abfangen und zur "richtigen" weiterleiten?

**Comment by theincogtion** [comment]
Übrigens das gleiche Problem gibts bei Schulz & Böhmermann.

**Comment by generia** [comment]
fixed with latest release 1.0.4
see also pull request https://github.com/xbmc/repo-plugins/issues/1244
